projects/storefrontapp-e2e-cypress/cypress/integration/regression/product-carousel/product-carousel.non-core-e2e-spec.ts

```diff
@@ -0,0 +1,39 @@
+import * as cart from '../../../helpers/cart';
```

Comment:
What I meant by non core is to rename it back to what it was before, which I presume was  `product-carousel.e2e-spec.ts`